postgresql-plugin/src/main/java/co/cask/postgres/PostgresSchemaReader.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.postgres;
+
+import co.cask.CommonSchemaReader;
+import co.cask.cdap.api.data.schema.Schema;
+import com.google.common.collect.ImmutableSet;
+
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.Set;
+
+/**
+ * PostgreSQL schema reader.
+ */
+public class PostgresSchemaReader extends CommonSchemaReader {
+
+  public static final Set<Integer> POSTGRES_TYPES = ImmutableSet.of(
+    Types.OTHER, Types.ARRAY, Types.SQLXML
+  );
+
+  public PostgresSchemaReader() {
```

Comment:
Seems that defining default constructor is unnecessary.

Reply:
done